### PR TITLE
refactoring

### PR DESCRIPTION
**Pull request by Borda**
some refactoring and code cleaning to have easier use without changing the code...
main features:
 - refactoring to package and scripts
 - the `yolo3` package can be installed
 - using scripts with parameters
 - updating some structures

### CODE REVIEW

**Comment by Borda** [comment]
@qqwweee could you have look, pls ^^

**Comment by Borda** [comment]
^^ @qqwweee

**Comment by Borda** [comment]
@qqwweee @philtrade @dleam @b02902131 is this repository still active?

File: scripts/annotation_coco.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    parser.add_argument('--path_images', type=str, required=True,
+                        help='Path to images of VOC dataset.')
+    parser.add_argument('--path_output', type=str, required=False, default='.',
+                        help='Path to output folder.')
+    arg_params = vars(parser.parse_args())
+    logging.debug('PARAMETERS: %s', repr(arg_params))
+    return arg_params
+
+
+def chenge_category(cat):
```
Comment (eherrerosj) [inline scripts/annotation_coco.py]: typo

File: scripts/annotation_csv.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
+"""
+Creating training file from costume dataset
```
Comment (eherrerosj) [inline scripts/annotation_csv.py]: typo

**Comment by Borda** [comment]
@qqwweee ^^

**Pull request closed without merging**